app/src/main/java/com/f0x1d/logfox/ui/activity/CrashDetailsActivity.kt

```diff
@@ -35,18 +38,22 @@ class CrashDetailsActivity: BaseViewModelActivity<CrashDetailsViewModel, Activit
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-
+        setTitle(R.string.crash_details)
         binding.scrollView.applyInsetter {
             type(navigationBars = true) {
                 padding(vertical = true)
             }
         }
 
         binding.toolbar.inflateMenu(R.menu.crash_details_menu)
-        binding.toolbar.setNavigationOnClickListener {
+        binding.toolbar.setupBackButton {
             onBackPressedDispatcher.onBackPressed()
         }
 
+        binding.copyLayout.setAccessibilityDelegateForClassName(Button::class.java)
+        binding.shareLayout.setAccessibilityDelegateForClassName(Button::class.java)
+        binding.zipLayout.setAccessibilityDelegateForClassName(Button::class.java)
```

Comment:
As I see, this method is only used with `Button` class. I guess you can make `setAccessibilityDelegateForButton` method in extensions